### PR TITLE
Fix for Spanish Translation + Add mspanish variant

### PR DESCRIPTION
**Pull request by PANCHO7532B**
So this is my attempt on fixing the current version + adding the mspanish (Mexican Spanish) variant as by default it falls back to English

Draft because:
1. A few game modes/match settings are still on English as I'm not really sure what kind of translation would be the adequate
2. I need feedback from other Spanish people so this can increase in quality and be "readable"
3. Spanish and Mexican Spanish are not usually the same so in my opinion I can't just re-use the same modism for the other

Por lo que si alguien habla Español, estoy abierto a sugerencias en este hilo para mejorar la traducción ^^
(O si no, la terminaré yó eventualmente/or else i'll finish it by myself eventually)

### CODE REVIEW

**Comment by PANCHO7532B** [comment]
Well, imho it's finished and ready to merge

**Comment by PANCHO7532B** [comment]
@BobTheBob9 pls merge :'v

File: Northstar.Client/mod/resource/northstar_client_localisation_mspanish.txt [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+
+		"SWITCH_YES" "Si"
+		"SWITCH_NO" "No"
+		"SWITCH_ANY" "Cualquiera"
+
+		"CONNECTING" "Conectando..."
+		"INGAME_PLAYERS" "Jugadores: ^6BA6C400%s1"
+		"TOTAL_SERVERS" "Servidores: ^C46C6C00%s1"
+
+
```
Comment (abarichello) [inline Northstar.Client/mod/resource/northstar_client_localisation_mspanish.txt:297]:
```suggestion

```
Remove this whitespace so it matches the number of lines we have in english.txt

**Review by abarichello — CHANGES_REQUESTED** [review]
spanish.txt is missing the `respawnprotection` key

File: Northstar.Client/mod/resource/northstar_client_localisation_mspanish.txt (hunk 1 above)
Comment (PANCHO7532B) [reply Northstar.Client/mod/resource/northstar_client_localisation_mspanish.txt]: Done

**Comment by PANCHO7532B** [comment]
Done @abarichello

**Review by abarichello — APPROVED** [review]
Thanks

**Pull request merged**